src/QMCWaveFunctions/einspline_spo.hpp

```diff
@@ -150,15 +150,16 @@ struct einspline_spo
       pos_type end(1);
       einsplines.resize(nBlocks);
       RandomGenerator<T> myrandom(11);
-      Array<T, 3> data(nx, ny, nz);
-      std::fill(data.begin(), data.end(), T());
-      myrandom.generate_uniform(data.data(), data.size());
       for (int i = 0; i < nBlocks; ++i)
       {
         einsplines[i] = myAllocator.createMultiBspline(T(0), start, end, ng, PERIODIC, nSplinesPerBlock);
         if (init_random)
-          for (int j = 0; j < nSplinesPerBlock; ++j)
-            myAllocator.set(data.data(), einsplines[i], j);
```

Comment:
I think we can remove all the set routines.

Reply:
Are we sure that some future use of the miniapp will not need to use realistic data and the set routines?  I'm hesitant to remove these routines right now.

Reply:
What the old routine (from Einspline) does is transforming the orbital value on a grid to spline coefficients. Even if we load realistic data, we load the coefficients directly and there is no need of transformation.

Reply:
I would prefer to do the removal in a separate PR.

Reply:
No problem.